### PR TITLE
Fixing Issue 278

### PR DESCRIPTION
**Pull request by Ryan-Thomas**
I've added the elements for the View Project page, but the GET API doesn't seem to be working (or maybe I'm just using it wrong) so I had to just leave some placeholder data for now.

Also, I need help with getting the label css to match the css of the other labels, because it looks kind of strange right now.

### CODE REVIEW

File: C4SGWeb/app/project/view/project-view.component.ts [hunk 1]
```diff
@@ -11,27 +12,50 @@ import { ProjectService } from '../common/project.service';
...
+    zip: 'zip',
+    country: 'country',
+  };
 
   constructor(private projectService: ProjectService,
               private route: ActivatedRoute,
               private router: Router,
-              private location: Location) {
+              private location: Location,
+              @Inject(FormBuilder) fb: FormBuilder) {
```
Comment (russon77) [inline C4SGWeb/app/project/view/project-view.component.ts]: why is `@Inject()` used here?

Comment (Ryan-Thomas) [inline C4SGWeb/app/project/view/project-view.component.ts]: Oh yeah that doesn't need to be there. It can be removed. Is there a way for me to delete that without submitting another pull request?

Comment (Code4SocialGood) [inline C4SGWeb/app/project/view/project-view.component.ts]: You can modify the code in your local repository, and make another commit.

Comment (Ryan-Thomas) [inline C4SGWeb/app/project/view/project-view.component.ts]: Yeah I fixed it

**Comment by russon77** [comment]
after accessing `http://localhost:3000/project/list` and selecting a project, i get this error in the console 

> Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'project/1'

i think you need to update the routes, i.e. `app-routing.module.ts`

**Comment by Ryan-Thomas** [comment]
When I access `http://localhost:3000/project/list` I get this error in the console:

> Failed to load resource: net::ERR_CONNECTION_REFUSED `http://localhost:8080/api/projects/all`

Do I need to do something first in order to make the API work?

**Comment by russon77** [comment]
@Ryan-Thomas i am curious how you tested the code that you wrote if you didn't have the backend running? that was the only way i could get the project list to display.

**Comment by Code4SocialGood** [comment]
Could you also resolve the conflicts?
